jsonschema/src/keywords/pattern_properties.rs

```diff
@@ -148,7 +150,7 @@ impl Validate for SingleValuePatternPropertiesValidator {
 impl ToString for SingleValuePatternPropertiesValidator {
     fn to_string(&self) -> String {
         format!(
-            "patternProperties: {{{}: {}}}",
+            "patternProperties: {{{:?}: {}}}",
```

Comment:
Is the format change needed?

Reply:
Same as above, this is required because `fancy_regex::Regex` implements `Debug` but not `Display`.